src/components/Blasts/BlastVisual.js

```diff
@@ -204,7 +204,10 @@ const styles = theme => ({
 function BlastVisual({ classes }) {
   function CurrentDate() {
     const date = new Date();
-    const today = date.toLocaleTimeString();
+    const today = date.toLocaleTimeString([], {
```

Comment:
Wasn't it decided that this should show the time of the last blast?

Reply:
@kilemensi 
Yap...but that is done in https://github.com/CodeForAfrica/sensors.AFRICA-SeaSensors/pull/54
Maybe we should merge that one first, then this one